### PR TITLE
[Mailer] ping_threshold for ses+smtp

### PR DESCRIPTION
**Pull request by Tyraelqp**
Docs for https://github.com/symfony/symfony/pull/42219

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -1112,6 +1119,8 @@ the final email)::
     $email->getHeaders()->addTextHeader('X-Transport', 'alternative');
     $mailer->send($email);
 
+.. _mailer-sending-messages-async:
```
Comment (OskarStark) [inline mailer.rst:1126]: Why this change?

File: mailer.rst [hunk 2]
```diff
@@ -174,6 +174,13 @@ party provider:
     For example, the DSN ``ses+smtp://ABC1234:abc+12/345@default`` should be
     configured as ``ses+smtp://ABC1234:abc%2B12%2F345@default``
 
+.. caution::
+
+    If you want to use `ses+smtp` transport together with :doc:`Messenger </messenger>`
+    to :ref:`send messages in background <mailer-sending-messages-async>`,
+    you need to add ``ping_threshold`` parameter to your ``MAILER_DSN`` with
+    value lower than ``10``: ``ses+smtp://USERNAME:PASSWORD@default?ping_threshold=9``
```
Comment (OskarStark) [inline mailer.rst:182]:
```suggestion
    value lower than ``10``: ``ses+smtp://USERNAME:PASSWORD@default?ping_threshold=9``

    .. versionadded:: 5.4
        
        The ping_threshold option was introduced in Symfony 5.4.
```

The option name please in double backticks, thanks

File: mailer.rst [hunk 3]
```diff
@@ -174,6 +174,13 @@ party provider:
     For example, the DSN ``ses+smtp://ABC1234:abc+12/345@default`` should be
     configured as ``ses+smtp://ABC1234:abc%2B12%2F345@default``
 
+.. caution::
+
+    If you want to use `ses+smtp` transport together with :doc:`Messenger </messenger>`
```
Comment (OskarStark) [inline mailer.rst]: Double backticks please

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: mailer.rst (hunk 1 above)
Comment (Tyraelqp) [reply mailer.rst:1126]: `ref` on line 180

File: mailer.rst (hunk 2 above)
Comment (Tyraelqp) [reply mailer.rst:182]: Added, ty

File: mailer.rst (hunk 3 above)
Comment (Tyraelqp) [reply mailer.rst]: Done

File: mailer.rst (hunk 1 above)
Comment (OskarStark) [reply mailer.rst:1126]: Indeed, sorry 😁

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Sergey, thanks a lot for contributing this feature and for contributing the docs too. It was a great first contribution to Symfony Docs! Congrats on that 🎉

**Pull request merged**